### DISCUSSION TITLE
Is this the place to talk about Voiceliner?

### DISCUSSION THREAD

**Discussion by faberfedor** [maxkrieger/voiceliner#50] [Q&A]
I _might_ have some ideas for enhancements, but I'm more interested in learning how to use this thing as in my workflow if possible. There are somethings I can't figure out.  Like what, you ask? Well...

1. How do you delete an outline? 
2. I'd like to take my voice notes on my Pixel 6 and save them at the end of my walk/session/day by dropping them into an Obsidian directory and have them processed. The process of Three Dots -> export markdown -> Select X-Plore to copy -> copy into a predefined directory with the default outline names is a bit...clunky. IS there a way to currently do this?
3. To take a note when I'm walking, I have to unlock my phone, bring Voiceliner to the front, and hold the on-screen button. Anyone know of a way to streamline this? I was thinking hitting some combinations of hardware buttons (any three of them!) to do all those steps but I've not found a way to do that.

Anyway, this app fills a need I didn't know I had, especially with the Obsidian angle. Looking forward to more developments.

**Comment by maxkrieger** [comment]
1. Archive it, and then the button will appear in the menu.

2. dunno, not really my own workflow. Feel free to fork / figure out a design that might work for you! I agree the current export markdown is kinda silly.

3. if an api existed I’d use it! Haven’t heard of any. Folks want Apple Watch support which seems like the right way of doing this. But I don’t own an Apple Watch.

**Comment by faberfedor** [comment]
1. That did it. Does it also delete the archive as well (I don't see any existence of it but...)
2. Initially I was thinking of forking it to add...something but then saw it was written in Flutter which I know absolutely nothing about.
3. I'll look around during my travels. Apple Watch, eh? Guess I should finally look into some Android watches. I think Google has some Pixel watches coming out this year. Maybe I should finally buy one. :-)

Anyone else out there with a Pixel watch or phone?
